pages/common/git-authors.md

```diff
@@ -0,0 +1,17 @@
+# git authors
+
+> Display log or list of committers of a Git repository. Everything except `--list` will create a new file called `AUTHORS`.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-authors>.
+
+- Display a log of committers in the default editor, reading from the `AUTHORS` file:
+
+`git authors`
+
+- Print a full list of committers:
+
+`git authors --list`
+
+- Display a log of committers, exluding emails, in the default editor, reading from the `AUTHORS` file:
```

Comment:
Why log?
```suggestion
- Display a full list of committers, excluding emails, in the default editor, reading from the `AUTHORS` file:
```
(a typo here)

Reply:
Good question. It is indeed only a plain list.

Reply:
< Why log? 

because the list is used in the list variant of the command.

Reply:
> because the list is used in the list variant of the command.

I didn't get this.

Reply:
But it's also a plain list when you run `git authors`.

Reply:
For me it opens in VIM, since it opens in the text editor

Reply:
--list makes a dumped list

Reply:
I think a `log` is a record of events, in this case were are not talking about events of any kind.

Reply:
But what does that have to do with a log? To me a log is something, where changes or events are saved with a timestamp, like `git log {{file}}`.

Reply:
To me a log, is a list of events as well, however it isn't required to be timestamped

Reply:
`git authors` only outputs a list of contributors, which I wouldn't call a log, but just a list.

Reply:
What happens exactly, the same as --list with it?

Reply:
`--list` don't create the `AUTHORS` file.